cocotb/share/makefiles/simulators/Makefile.icarus

```diff
@@ -69,26 +67,25 @@ LIB_LOAD = PATH=$(OLD_PATH):$(LIB_DIR)
 NEW_PYTHONPATH:=$(shell python -c "import sys, os; print(':'.join(['/'+dir.replace(os.sep,'/').replace(':','') for dir in sys.path]))")
 
 else
-LIB_LOAD := LD_LIBRARY_PATH=$(LIB_DIR):$(LD_LIBRARY_PATH)
 NEW_PYTHONPATH := $(PYTHONPATH)
 endif
 
 $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/sim.vvp $(CUSTOM_SIM_DEPS) $(COCOTB_LIBS) $(COCOTB_VPI_LIB)
 	-@rm -f $(COCOTB_RESULTS_FILE)
 
-	PYTHONPATH=$(LIB_DIR):$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
+	PYTHONPATH=$(PWD):$(PYTHONPATH):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
```

Comment:
```suggestion
	PYTHONPATH=$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
```
Looks like an obvious typo

Reply:
This was before and did not wok. It is some combination of `;` and `:` on windows etc. It should work now although I would prefer #1462.